### ISSUE TITLE
Where are the versions?

### ISSUE THREAD

**Issue by xmiseggs** [k9mail/k-9#3995]
I don't get it. What am I missing? I see that they are fixing bugs and adding improvements because they are making changes almost daily, but I continue with the version of September 7, 2018 Why? Who is lucky enough to try the application? What has been done in these 8 months is not important to publish a new version?

**Comment by Revertron** [comment]
Yep, I wonder too!

**Comment by mruwek** [comment]
@cketti recently [wrote]( https://github.com/k9mail/k-9/issues/3940#issuecomment-470811966) that he don't want end-users to try development builds.

You can always compile it locally - that's what I've done, but I found that swipe-to-select stopped working, so I reverted to 5.600

**Comment by gene-git** [comment]
I compiled in android studio (signed apk) and tested in simulator - but, after allowing non trusted source on Android 9 pie, it would not install - I suspect I need to disable google protect - (didn't try that yet- and I'm loathe to leave that permanently) - can i keep google protect and install the self signed apk?

**Comment by xmiseggs** [comment]
When is the next version due?

**Comment by gene-git** [comment]
I will go back to trying to install a private build - runs fine in android studio simulator - any suggestions on how to install a private signed apk? Do i need to remove the official version first and then install my build? 

Not that this is at all reasonable but 5.8 is at 60%, so linear extrapolation would put release date sometime jan 2020. Obviously a linear extrap is plain silly but there you go ... :)

Back to android studio ...

**Comment by gene-git** [comment]
I can build the git version and run it fine in Android Studio simulator - copying signed apk, sadly, no luck - i have not tried installing via adb yet. That said 5.8 is looking like it will be a decent release eventually some day. Best I can tell in the simulator its working quite well now. Clearly more to do but pretty nice. Way better than 5.6.

Shame that k9 is a bit slow and hard (for me anyway) to test.  While I understand the desire for non-tech folks not to generate noise with test builds, I found the friction too high. I would encourage the devs to offer at least some minimal guidance on how to test a private build and to start offering test/beta builds soon.

However I've also started testing other apps - FairEmail looks like a pretty good alternative. Has openpgp/autocrypt, seems to works properly with android doze / 9+ and uses material design. It costs a small amount to unlock pgp but seems pretty worth it to me.

I'm sure k9mail will get there eventually, and I look forward to trying it again when I can run a newer version.
